modules/javafx.controls/src/main/java/javafx/scene/control/TableView.java

```diff
@@ -395,6 +395,92 @@
         }
     };
 
+    /**
+     * A policy that tries to adjust other columns in order to fit the table width.
+     * <p>
+     * During UI adjustment, proportionately resizes all columns.
```

Comment:
You might want to add `to preserve the total width` here since you say it for some of the other policies.